### ISSUE TITLE
Data file structure

### ISSUE THREAD

**Issue by j-faria** [j-faria/kima#52]
The order of the columns in the data file should always be

    time   RV   RVerr   FWHM   BIS   R'hk   R'hkerr

right, @jdavidrcamacho ?

In terms of units, FWHM and BIS are (usually) in the same units as RV and RVerr. I think it's safe to assume that. Once this is implemented, we need to 

- [ ] write documentation

**Comment by jdavidrcamacho** [comment]
Yes! And FWHM and BIS will be multiplied by 1000 every time we set the unit to m/s just like the RVs.

I'll start writing documentation soon
